selfdrive/sensord/tests/test_sensord.py

```diff
@@ -116,10 +116,14 @@ def setUpClass(cls):
     # make sure gpiochip0 is readable
     HARDWARE.initialize_hardware()
 
-  @with_processes(['sensord'])
+    # read initial sensor values every test case can use
+    managed_processes["sensord"].start()
+    cls.events = read_sensor_events(3)
```

Comment:
```suggestion
    cls.events = read_sensor_events(5)
```

Reply:
gonna need to do a quick test run with that change, check if it doesnt screw with the values